File: public/examples/integrations/mcp-servers/engine_api/check_arcade_engine_health_example_call_tool.py
import json
from arcadepy import Arcade

client = Arcade()  # Automatically finds the `ARCADE_API_KEY` env variable

TOOL_NAME = "EngineApi.CheckArcadeEngineHealth"

tool_input = {
    
}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
)
print(json.dumps(response.output.value, indent=2))

File: public/examples/integrations/mcp-servers/engine_api/check_arcade_engine_health_example_call_tool.js
import { Arcade } from "@arcadeai/arcadejs";

const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable

const TOOL_NAME = "EngineApi.CheckArcadeEngineHealth";

const toolInput = {};

const response = await client.tools.execute({
  tool_name: TOOL_NAME,
  input: toolInput,
});

console.log(JSON.stringify(response.output.value, null, 2));
